### ISSUE TITLE
Vuejs component library use-case

### ISSUE THREAD

**Issue by matschik** [Developmint/vue-link#49]
Hello !

Just want to know if `vue-link` main purpose is for Vuejs component library dealing with HTML links.
For example, when I import my <Menu/> component (integrating `vue-link`) in my Nuxtjs app, it "transforms" to <nuxt-link/> right ?
If I import it in any Vuejs app using 'vue-router', it "transforms" to <router-link/> ?

If so, that's awesome !! Exactly what I need !

**Comment by manniL** [comment]
Nope, that unfortunately won't work right now but would be a nice enhancement 😋
